src/ContactPage/Contact.scss

```diff
@@ -0,0 +1,68 @@
+@import "../styles/typography";
+
+.contact-section {
+  height: 100vh;
+
+  &__border {
+    border: none;
+    padding: 2% 0;
+    background: rgb(229, 226, 226);
+  }
+  &__title {
+    font-family: $base-font-family;
+    margin: 0 5%;
+    margin-bottom: 10px;
+  }
+
+  &__form {
+    margin: 0 5%;
+    display: flex;
```

Comment:
Why is this required? Won't inputs stack ontop of each other automatically anyway?